src/experiments/OOD-adj-scores/0_Train_Models.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
Maybe let's avoid abbreviation in the folder name, as it may be clearer to call this folder: `adjusted-OOD-scores`?

Reply:
Also update main Readme with a brief description of what's inside of this folder?  You could alternatively have a Readme inside of this folder with a couple sentences about the adjusted scores and additional imbalanced benchmark setting.